### PR TITLE
:bug: fix MucDivider

### PR DESCRIPTION
**Pull request by markostreich**
**Description**
I tried to fix the mentioned bug in the recurring issue since I need it.

**Reference**

Issues #183

### CODE REVIEW

File: public/assets/temporary/custom-style.css [hunk 1]
```diff
@@ -67,5 +67,5 @@ h5 {
 .muc-divider {
     align-self: stretch;
     height: 0;
-    border: 1px var(--color-neutrals-blue) solid;
+    border-bottom: 1px var(--color-neutrals-blue) solid;
 }
```
Comment (langehm) [inline public/assets/temporary/custom-style.css:71]: We should outsource this css-class into a css-file inside the muc-divider folder or into the component itself.

If this css class isn't used by any other component the later way would be prefered.

File: src/components/Divider/MucDivider.vue [hunk 2]
```diff
@@ -1,9 +1,3 @@
 <template>
-  <hr class="divider-border" />
+  <div class="muc-divider" />
```
Comment (langehm) [inline src/components/Divider/MucDivider.vue:2]: We need to help screenreaders with the empty div ... [Comment](https://github.com/it-at-m/muc-patternlab-vue/issues/183#issuecomment-2428720690)

Comment (FabianWilms) [reply src/components/Divider/MucDivider.vue:2]: In which way do we need to help screenreaders with this?

Most screenreaders ignore hr-Tags all together and empty divs are handled the same way.

Comment (langehm) [reply src/components/Divider/MucDivider.vue:2]: The `hr` tag define when a new section starts or ends. This tag will is will be read as a separator.
We can postpone this into another issue

File: public/assets/temporary/custom-style.css (hunk 1 above)
Comment (markostreich) [reply public/assets/temporary/custom-style.css:71]: It is used by two more components (MucCard, MucIntro).
As our design coordinator mentioned, the height of the divider should be 1px in all these instances.

Comment (langehm) [reply public/assets/temporary/custom-style.css]: With [this](https://github.com/it-at-m/muc-patternlab-vue/pull/282/commits/dd7317cef8da72df81402ae7fd6af4c1c00b04cd) commit, they do not use it any more. If we already have the component, we can use in other components too.

I only found those two occurrences.

**Comment by langehm** [comment]
@FabianWilms please check this PR. I committed code to this PRand therefore don't want to self approve this 😉

**Review by FabianWilms — APPROVED** [review]

**Pull request merged**